### PR TITLE
Ui

### PR DESCRIPTION
**Pull request by stetskivostap**

### CODE REVIEW

File: VirtualClassRoom/ClientApp/src/app/services/AuthenticationService.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import { BehaviorSubject, Observable } from "rxjs";
+import { LoginResponse } from "../Models/LoginResponse";
+import { Injectable } from "@angular/core";
+
+@Injectable({
+    providedIn: 'root'
+})
+
+export class Auth{   
+    public Token: string;
```
Comment (vladpavlyuk) [inline VirtualClassRoom/ClientApp/src/app/services/AuthenticationService.ts:13]: In JS or TS (TypeScript) use lowercase for variable naming

File: VirtualClassRoom/ClientApp/src/app/services/AuthenticationService.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        return this.http.post<any>(`${environment.apiUrl}/api/users/login`, { Email, password })
+            .pipe(map(token => {
+                localStorage.setItem('Token', JSON.stringify(token));                              
+                return token;
+            }));
+            
+    }
+
+    logout() {       
+        localStorage.removeItem('Token');
```
Comment (vladpavlyuk) [inline VirtualClassRoom/ClientApp/src/app/services/AuthenticationService.ts:27]: Use lowercase for Token

File: VirtualClassRoom/ClientApp/src/app/login-form/login-form/login-form.component.scss [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+ul {
+  list-style: none;
+}
+
+img {
+  width: 100%;
+}
+
+.btn {
```
Comment (vladpavlyuk) [inline VirtualClassRoom/ClientApp/src/app/login-form/login-form/login-form.component.scss:39]: You can use Bootstrap _btn_ class, but it’s not required

**Comment by stetskivostap** [comment]
About  registration bug, it was`nt in that project? i misclicked(

File: VirtualClassRoom/ClientApp/src/environments/environment.ts [hunk 4]
```diff
@@ -3,7 +3,8 @@
 // The list of file replacements can be found in `angular.json`.
 
 export const environment = {
-  production: false
+  production: false,
+  apiUrl: 'https://localhost:44354',
```
Comment (vladpavlyuk) [inline VirtualClassRoom/ClientApp/src/environments/environment.ts:7]: Do not use _api_ prefix for fronted url

File: VirtualClassRoom/ClientApp/src/app/nav-menu/nav-menu.component.html [hunk 5]
```diff
@@ -17,26 +17,14 @@
...
-              >Counter</a
-            >
-          </li>
+        [ngClass]="{ show: isExpanded }">
+      
+        <ul class="navbar-nav flex-grow">          
           <li class="nav-item" [routerLinkActive]="['link-active']">
-            <a class="nav-link text-dark" [routerLink]="['/fetch-data']"
-              >Fetch data</a
+            <a class="nav-link text-dark" [routerLink]="['/api/login']"
```
Comment (vladpavlyuk) [inline VirtualClassRoom/ClientApp/src/app/nav-menu/nav-menu.component.html:24]: Remove _api_ prefix

File: VirtualClassRoom/ClientApp/src/app/login-form/login-form/login-form.component.html [hunk 6]
```diff
@@ -0,0 +1,23 @@
+<div class="container">
+    <form class="card" [formGroup]="form">
+      <h1>Login</h1>
+  
+      <div class="form-control">
+        <label>Email</label>
+        <input type="text" placeholder="email" formControlName="email" required>
```
Comment (vladpavlyuk) [inline VirtualClassRoom/ClientApp/src/app/login-form/login-form/login-form.component.html:7]: 1) Add Regular Expression validation for email. Show error message if email isn't match this expression.
2) If email input is empty then make input border color as red and show an error message below this input field,

File: VirtualClassRoom/ClientApp/src/app/login-form/login-form/login-form.component.html [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+      </div>
+      <hr/>
+      <hr/>
+      <hr/>
+      <div class="form-control">
+        <label>Password</label>
+        <input type="password" placeholder="password" formControlName="password" required>
```
Comment (vladpavlyuk) [inline VirtualClassRoom/ClientApp/src/app/login-form/login-form/login-form.component.html:14]: 1. Set a minimum length as 6 symbol.
2. Password should contain at least 1 digit symbol
3. Password should contain at least 1 special character
4. Password should contain at least 1 upper-case letter
5. If password doesn't match at least one of the rules that I mentioned above then show an error message with these rules
6.  If password input is empty then make input border color as red and show an error message below this input field

File: VirtualClassRoom/ClientApp/src/app/services/AuthenticationService.ts [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+    providedIn: 'root'
+})
+
+export class Auth{   
+    public token: string;
+    constructor(private http: HttpClient){        
+    }   
+
+    login(Email: string, password: string) {
+        return this.http.post<any>(`${environment.apiUrl}/api/users/login`, { Email, password })
```
Comment (vladpavlyuk) [inline VirtualClassRoom/ClientApp/src/app/services/AuthenticationService.ts:18]: Investigate this issue. Property names should be in the same case

**Pull request merged**